packages/shared/src/hooks/useFeedSettings.ts

```diff
@@ -53,14 +54,31 @@ export default function useFeedSettings({
   );
 
   const { tagsCategories, feedSettings, advancedSettings } = feedQuery;
+  const checkSettingsEnabledState = useCallback(
+    (idParam: number) => {
+      const advancedSetting = advancedSettings?.find(
+        ({ id }) => id === idParam,
+      );
+
+      if (!advancedSettings) {
```

Comment:
I think this should be 
```suggestion
      if (!advancedSetting) {
```

Reply:
Oh, good catch! 🚀

Reply:
Maybe a good sign that I should rename the variable name entirely to void confusion.